{
    "config": {
        "abort": {
            "already_configured": "Urz\u0105dzenie jest ju\u017c skonfigurowane",
            "cannot_connect": "Nie mo\u017cna nawi\u0105za\u0107 po\u0142\u0105czenia z odnalezionym Volumio"
        },
        "error": {
            "cannot_connect": "Nie mo\u017cna nawi\u0105za\u0107 po\u0142\u0105czenia",
            "unknown": "Nieoczekiwany b\u0142\u0105d"
        },
        "step": {
            "discovery_confirm": {
                "description": "Czy chcesz doda\u0107 Volumio (\"{name}\") do Home Assistanta?",
                "title": "Wykryto Volumio"
            },
            "user": {
                "data": {
                    "host": "Host",
                    "port": "Port"
                }
            }
        }
    }
}